app/src/androidTest/java/org/projectbuendia/client/ui/chart/PatientChartActivityTest.java

```diff
@@ -51,6 +58,9 @@
     private static final String YES = "●";
 
     private static final int ROW_HEIGHT = 84;
+    private static final Matcher<View> OVERFLOW_BUTTON_MATCHER = anyOf(
+            allOf(isDisplayed(), withContentDescription("More options")),
+            allOf(isDisplayed(), withClassName(endsWith("OverflowMenuButton"))));;
 
```

Comment:
Actually I think this is ok. We use 8 spaces for line continuation. See https://github.com/projectbuendia/buendia/wiki/Android-Style-Guide#452-continuation-indentation-at-least-8-spaces